README.md

```diff
@@ -1,80 +1,74 @@
-# [![Learn Anything](media/header.png)](https://learn-anything.xyz/)
+# Learn Anything
 
-# Learn Anything [![Discord](https://img.shields.io/badge/chat-Discord-blue.svg)](https://discord.gg/KKYdWjt) [![Twitter](https://img.shields.io/twitter/follow/learnanything_.svg?style=social&label=Follow&style=flat-square)](https://twitter.com/learnanything_) [![Support](https://img.shields.io/badge/Support%20Us-💗-ff69b4.svg)](https://www.patreon.com/learnanything) [![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/learn-anything)
+[![Discord](https://img.shields.io/badge/-Discord-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://discord.gg/KKYdWjt)
+[![Twitter](https://img.shields.io/badge/-Twitter-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://twitter.com/learnanything_)
+[![Spectrum](https://img.shields.io/badge/-Spectrum-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://spectrum.chat/learn-anything)
+[![Paper](https://img.shields.io/badge/-Paper-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://github.com/learn-anything/learn-anything/wiki/White-Paper)
+[![Support](https://img.shields.io/badge/%E2%9D%A4-Support-0a0a0a.svg?style=flat&colorA=0a0a0a)](https://www.patreon.com/learnanything)
 
-[Learn Anything](https://learn-anything.xyz/) is an open source website to organize world's knowledge with study plans for learning anything.
+#### [Learn Anything](https://learn-anything.xyz) is a project to organize world's knowledge in a mind map and curate study plans for learning anything.
```

Comment:
I don't think mind map is the correct term to use. Ideally topics will be represented in a graph way. But for explore there can be many ways to represent and visualize topics.

-- 

[Learn Anything](https://learn-anything.xyz) is a project to organize world's knowledge in a visual way with study plans for learning anything.

--

Do you think above is better?